build_web_compilers/pubspec.yaml

```
@@ -1,10 +1,10 @@
 name: build_web_compilers
-version: 4.0.6
+version: 4.0.7
 description: Builder implementations wrapping the dart2js and DDC compilers.
 repository: https://github.com/dart-lang/build/tree/master/build_web_compilers
 
 environment:
-  sdk: '>=3.0.0 <3.4.0'
+  sdk: ^3.1.0
```

This broadens the upper constraint, but we have an agreement not to do that with the web compiler teams so that the command line APIs can evolve:

```suggestion
  sdk: '>=3.1.0 <3.4.0'
```